src/unit-tests/osfile-test/ut_osfile_test.c

```diff
@@ -78,18 +78,17 @@ int32 UT_os_setup_fs()
     if (res != OS_SUCCESS)
     {
         UtPrintf("OS_mkfs() returns %d\n", (int)res);
-        goto UT_os_setup_fs_exit_tag;
+        return res;
     }
 
     res = OS_mount(g_devName, g_mntName);
     if (res != OS_SUCCESS)
     {
         UtPrintf("OS_mount() returns %d\n", (int)res);
         OS_rmfs(g_devName);
-        goto UT_os_setup_fs_exit_tag;
+        return res;
     }
 
-UT_os_setup_fs_exit_tag:
     return res;
```

Comment:
Multiple returns should be avoided.  Consider and if/else and remove the second return:
```
if
{
  print
}
else
{
  mount
  if
  {
     print and rmfs
  }
}
return res
```